app/cmd.js

```diff
@@ -13,10 +13,12 @@ var sortJson = require('./');
 var files = process.argv.slice(0).filter(arg => arg.endsWith('.json') || arg.endsWith('.rc'));
 var ignoreCase = _.includes(process.argv, '--ignore-case') || _.includes(process.argv, '-i');
 var reverse = _.includes(process.argv, '--reverse') || _.includes(process.argv, '-r');
+var dirtyDepth = process.argv.slice(0).filter(arg => arg.startsWith('-d') || arg.startsWith('--depth'));
+var depth = dirtyDepth.length > 0 ? dirtyDepth[0].split('=')[1] : false;
```

Comment:
Let's do a `parseInt(dirtyDepth[0].split('=')[1], 10) here just so we're sure we're getting a Number

Reply:
Of course! I'll do that.